docs/src/loo_compare.md

```diff
@@ -34,52 +33,46 @@ data = rand(dist, 1000)
 
 The following code block defines the model along with its prior distributions using Turing.jl. We'll use this model with different fixed values for the `k` parameter.
 
-```@example loo_compare
-@model function model_LBA(data, k; min_rt=0.2)
+```@example loo_example
+@model function model_LBA(data, k; min_rt=minimum(data.rt))
     # Priors
     ν ~ MvNormal(fill(3.0, 2), I * 2)
     A ~ truncated(Normal(0.8, 0.4), 0.0, Inf)
     τ ~ Uniform(0.0, min_rt)
 
     # Likelihood
-    for i in 1:length(data)
-        data[i] ~ LBA(; ν, A, k, τ)
-    end
+    data ~ LBA(; ν, A, k, τ)
```

Comment:
note that this syntax will prevent the subsequent from getting standard errors. Which is why I used the former
see: 

https://turinglang.org/ParetoSmooth.jl/stable/turing/#Incorrect-Model-Specification

Reply:
That's right. We need the for loop for the pointwise loglikelihood. Thanks for reminding me. I will submit a fix.